thirdparty/s3.go

```diff
@@ -47,9 +49,10 @@ func PreSign(r RequestParams) (string, error) {
 	return urlStr, err
 }
 
+// GetHeadObject fetches the metadata of an s3 object
 func GetHeadObject(r RequestParams) (*s3.HeadObjectOutput, error) {
 	session, err := session.NewSession(&aws.Config{
-		Region: aws.String(endpoints.UsEast1RegionID),
+		Region: &r.Region,
```

Comment:
nit: to make this consistent when using the AWS SDK, can we use `aws.String(r.Region)`?